src/translations.js

```diff
@@ -240,6 +248,7 @@ const langCardLocales = {
     sk: "Najviac používané jazyky",
     tr: "En Çok Kullanılan Diller",
     pl: "Najczęściej używane języki",
+    vi: "Ngôn Ngữ Thường Sử Dụng",
```

Comment:
Similar here, google translate gives `Các ngôn ngữ được sử dụng phổ biến` which is probably wrong but just double-checking.

Reply:
- `Phổ biến` is a word that refers to something that is commonly used by people, And I think it's not needed in this case
- `Thường sử dụng` to refer to something commonly used by 1 person, I think this word is more literal